rest-backend/src/main/java/com/sap/psr/vulas/backend/rest/BugController.java

```diff
@@ -301,10 +301,10 @@
 	public ResponseEntity<List<AffectedLibrary>> addAffectedLibrary(@PathVariable String bugid, @RequestParam(value="source", required=true) AffectedVersionSource source, @RequestBody AffectedLibrary[] affectedLibraries) {
 		//Ensure that PUT is only used by CHECK_VERSION or PATCH EVAL results
 		final StopWatch sw = new StopWatch("PUT affected libraries for source: " + source).start();
-		if(!source.equals(AffectedVersionSource.CHECK_VERSION)&&!source.equals(AffectedVersionSource.AST_EQUALITY)&&!source.equals(AffectedVersionSource.GREATER_RELEASE)&&!source.equals(AffectedVersionSource.INTERSECTION)&&!source.equals(AffectedVersionSource.MINOR_EQUALITY)&&!source.equals(AffectedVersionSource.MAJOR_EQUALITY)&&!source.equals(AffectedVersionSource.TO_REVIEW) &&!source.equals(AffectedVersionSource.PROPAGATE_MANUAL)){
```

Comment:
Disabling this check means that affected lib info can be PUT no matter the source